src/services/RetroGenerationService.ts

```diff
@@ -365,9 +377,14 @@ class RetroGenerationService {
    * @param testType
    */
   private isPassingLECTestType(testType: any): boolean {
-    const lecTestTypeIds = ["39", "44", "45"];
-    return lecTestTypeIds.includes(testType.testTypeId) && testType.testResult === TEST_RESULT_STATES.PASS;
-  }
+        const lecTestTypeIds = ["39", "44", "45"];
+        return lecTestTypeIds.includes(testType.testTypeId) && testType.testResult === TEST_RESULT_STATES.PASS;
+    }
+
```

Comment:
method docs missing

Reply:
I believe you where referring to isTestTypeCoifWithAnnualTestOrCoifWithAnnualTestRetest() method. fixed